src/main/java/ua/ithillel/lesson15/ProductMoving.java

```diff
@@ -0,0 +1,88 @@
+package ua.ithillel.lesson15;
+
+import java.time.LocalDate;
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class ProductMoving {
+
+    private final List<Product> list = new ArrayList<>();
+
+    public void add(Product product){
+        list.add(product);
+    }
+
+    public List<Product> getWholeList(){
```

Comment:
```suggestion
    public List<Product> getAll(){
```
просто незвична назва для метода. інші альтернативи: getItems, getProducs